_docs/1.0.0/usage-schema.md

```diff
@@ -0,0 +1,618 @@
+| `∞`    | *not part of pattern* | String ti represent infinity                                                                                                                                        |                                                                                    |
+
+<a name="pattern-parsing-star"/>\* while these can ne changed, in the [`pattern`]($pattern) the default value *must* be 
+used. E.g. *"."*, *"."* and *"-"*.
+
+**NB!** If there's no special format of the input, it's advised to avoid the usage of patterns due to the added
+performance hit (parsing using a pattern adds computational overhead). It's still possible to redefine the special
+symbols like [decimal separator](#decimal_separator) or [minus sign](#minus_sign) even without the pattern.
+
+#### Number parsing peculiarities
+
+- number of `#` and `0` in pattern is not reliable, stuff is parsed regardless (e.g. *"0.0"* produces the same result as
+*"##.#*)
+- Grouping signs placement "flexible"; in other words if grouping separator has been added to the pattern the number is
+parser regardless of the position (e.g. pattern `#,##0.#` will parse _1,000.0_, _1,1234.5_ so as  _2000._)
+- Grouping sign not accepted in decimal places
+- _"+"_ not accepted if pattern is specified unless explicitly mentioned
+- `%` and `‰` in fractional and decimal values and patterns divide the number by 100 and 1000 (in integrals as well, but
+then they are usually not whole numbers)
+- Even if redefined  the standard grouping and decimal separators **needs to be used** in the pattern
+- If pattern is used, `e` is not accepted only `E` in the exponential expresion (without a pattern both are recognized)
+
+Defaults
+--------
+
+As it was described, in case when the standardization is unsuccessful, either due to missing data in a column
+non-nullable column or because the casting fails, default value is used in that particular column and row.
+
+### Explicit default
+
+The value optionally specified in the schema of the dataset for **Standardization** in _metadata_ [Default](#default)
+property to be used for the particular column.
+
+### Global default values
+
+The value used in case of need of the default value and if _explicit default_ was not defined. The actual values are:
+
+- `null` for nullable column
+- `0` for numeric column
+- `false` for Boolean column
+- `""` (empty string) for string column
+- `1970/01/01` for date column
+- `1970/01/01 00:00:00` for timestamp column
+
+- default timezone if not specified for both timestamps and dates is *UTC*; this can be changed via
+- default locale and the decimal symbols based on that is *US*. Therefore:
+  - minus sign is *"-"*
+  - decimal separator is *"."*
+  - grouping separator is *","*
+
+### Explicit default values restrictions
+
+- The value has to be a string convertible to the the field's type and fitting withing its size limitations (e.g._"200"_
```

Comment:
```suggestion
- The value has to be a string convertible to the field's type and fitting within its size limitations (e.g._"200"_
```

Reply:
This is the only one you missed @benedeki